.circleci/config.yml

```diff
@@ -407,10 +407,19 @@ workflows:
   version: 2
   build_and_test:
     jobs:
+      - publish-docker-build
       - unit_tests
+          requires:
+            - publish-docker-build
       - cdlang_tests
+          requires:
+            - publish-docker-build
       - coverage
+          requires:
+            - publish-docker-build
       - lint-and-style-checks
+          requires:
+            - publish-docker-build
       - license-check
       - markdown-style-check
   docker-publish:
```

Comment:
We still want to publish the builder image with `stratumprojct/build:build` in post-merge job